ext/TestExt/Rings-conformance-tests.jl

```diff
@@ -390,6 +390,7 @@ function test_EuclideanRing_interface(R::AbstractAlgebra.Ring; reps = 20)
          test_mutating_op_like_add(mod, mod!, f, m)
          test_mutating_op_like_add(gcd, gcd!, f, m)
          test_mutating_op_like_add(lcm, lcm!, f, m)
+         test_mutating_op_like_add(divides, divides!, f, m; only3arg = true)
```

Comment:
```suggestion
         test_mutating_op_like_add(divides, divides!, f, f*m; only3arg = true)
         test_mutating_op_like_add(divides, divides!, f, m; only3arg = true)
```
Both cases would be interesting